content/php/concepts/math-functions/terms/atan/atan.md

```diff
@@ -0,0 +1,76 @@
+---
+Title: 'atan()'
+Description: 'Returns the angle whose tangent value is equal to the specified value.'
+Subjects:
+  - 'Web Development'
+  - 'Code Foundations'
+  - 'Computer Science'
+  - 'Math Functions'
```

Comment:
```suggestion
```
Not a subject in [the subjects.md file](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md).